streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -52,6 +53,8 @@
     final Logger log;
     final LogContext logContext;
     final StateDirectory stateDirectory;
+    // These are source partitions that also serve as a changelog for a store.
+    protected final Set<TopicPartition> sourceChangelogPartitions;
```

Comment:
Rename to something like `sourcePartitionsAsChangelog` and remove comment above.